docs/source/admin.rst

```diff
@@ -74,6 +74,12 @@ created
   ``linkreference``, ``testcases``, ``testplans`` and ``testruns``
   applications.
 
+To ensure that all necessary permissions are assigned to the default group (Tester)
+and that the stale permission objects are removed, after the system installation
+or upgrade use the following management command::
+
+    ./manage.py refresh_permissions
+
```

Comment:
Found out we try to assign permissions to groups in `core/migrations/0001_squashed.py` which kind-of doesn't work b/c the permission objects are probably not created at this point. Leaving this comment so we don't forget to deal with it.